TerminalDocs/customize-settings/global-settings.md

```diff
@@ -0,0 +1,275 @@
+
+:::column-end:::
+:::row-end:::
+
+<br />
+
+___
+
+## Tab settings
+
+### Always show tabs
+
+:::row:::
+:::column span="":::
+When set to `true`, tabs are always displayed. When set to `false` and `showTabsInTitlebar` is set to `false`, tabs only appear after more than one tab exists by typing `ctrl+shift+t`.
+
+**Property name:** `alwaysShowTabs`
+
+**Necessity:** Required
+
+**Accepts:** `true`, `false`
+
+**Default value:** `true`
+
+:::column-end:::
+:::column span="":::
+![Windows Terminal always show tabs](./../images/always-show-tabs.gif)
+
+:::column-end:::
+:::row-end:::
+
+### Tab width mode
+
+:::row:::
+:::column span="":::
+Sets the width of the tabs. `"equal"` makes each tab the same width. `"titleLength"` sizes each tab to the length of its title.
+
+**Property name:** `tabWidthMode`
+
+**Necessity:** Optional
+
+**Accepts:** `"equal"`, `"titleLength"`
+
+**Default value:** `"equal"`
+
+:::column-end:::
+:::column span="":::
+![Windows Terminal tab width mode](./../images/tab-width-mode.gif)
+
+:::column-end:::
+:::row-end:::
+
+### Hide close all tabs popup
+
+:::row:::
+:::column span="":::
+When set to `true`, closing a window with multiple tabs open _will_ require confirmation. When set to `false`, closing a window with multiple tabs open _will not_ require confirmation.
+
+**Property name:** `confirmCloseAllTabs`
+
+**Necessity:** Optional
+
+**Accepts:** `true`, `false`
+
+**Default value:** `true`
+
+:::column-end:::
+:::column span="":::
+![Windows Terminal confirm close all tabs](./../images/confirm-close-all-tabs.png)
+
+:::column-end:::
+:::row-end:::
+
+<br />
+
+___
+
+## Launch settings
+
+### Launch maximized
+
+Defines whether the Terminal will launch as maximized or not.
+
+**Property name:** `launchMode`
+
+**Necessity:** Optional
+
+**Accepts:** `"default"`, `"maximized"`
+
+**Default value:** `"default"`
+
+### Launch position
+
+The position of the top left corner of the window upon first load. On a system with multiple displays, these coordinates are relative to the top left of the primary display. If `launchMode` is set to `"maximized"`, the window will be maximized on the monitor specified by those coordinates.
+
+**Property name:** `initialPosition`
+
+**Necessity:** Optional
+
+**Accepts:** Coordinates as a string
+
+**Default value:** `","`
+
+### Columns on first launch
+
+The number of columns displayed in the window upon first load.
+
+**Property name:** `initialCols`
+
+**Necessity:** Required
+
+**Accepts:** Integer
+
+**Default value:** `120`
+
+### Rows on first launch
+
+The number of rows displayed in the window upon first load.
+
+**Property name:** `initialRows`
+
+**Necessity:** Required
+
+**Accepts:** Integer
+
+**Default value:** `30`
+
+<br />
+
+___
+
+## Title bar settings
+
+### Show/Hide the title bar
+
+:::row:::
+:::column span="":::
+When set to `true`, the tabs are moved into the title bar and the title bar disappears. When set to `false`, the title bar sits above the tabs.
```

Comment:
So, just throwing the idea out there again, what do you think of moving these descriptions of true/false to "accepts" and providing more of an overview description up here.

Same with `showTerminalTitleInTitlebar` below